bots/csr/src/test/java/org/openjdk/skara/bots/csr/CSRBotTests.java

```diff
@@ -429,49 +443,57 @@ void testCsrUpdateMarker(TestInfo testInfo) throws IOException {
             var editHash = CheckableRepository.appendAndCommit(localRepo);
             localRepo.push(editHash, repo.url(), "edit", true);
             var pr = credentials.createPullRequest(repo, "master", "edit", issue.id() + ": This is an issue");
+            // Add the notification link to the PR in the issue. This is needed for the CSRIssueBot to
+            // be able to trigger on CSR issue updates
+            PullRequestUtils.postPullRequestLinkComment(issue, pr);
```

Comment:
Just a word of appreciation. It is great that you document so clearly all these testing steps. It makes all the difference in understanding what the test is trying to do, and why it should succeed.